src/DataCollector/QueryCollector.php

```diff
@@ -378,7 +378,7 @@ protected function findMiddlewareFromFile($file)
         $filename = pathinfo($file, PATHINFO_FILENAME);
 
         foreach ($this->middleware as $alias => $class) {
-            if (strpos($class, $filename) !== false) {
+            if (!is_null($class) && !is_null($filename) && strpos($class, $filename) !== false) {
```

Comment:
Couldn't it be something simpler? like
```php
if ($class && strpos($class, (string) $filename) !== false) {
``` 
```suggestion
            if ($class && strpos($class, (string) $filename) !== false) {
```